raiden_contracts/tests/README.md

```diff
@@ -0,0 +1,57 @@
+# Raiden Network Smart Contracts Tests
+
+
+## Current `raiden-contracts` structure:
+
+- raiden_contracts
+    - contracts
+        - lib - libraries used by core contracts
+        - services - containes 3rd party services contracts
+        - test - test contracts used to test core contracts
+        - raiden core contracts files
+    - data - compiled contracts data & deployment information
+    - data_pre_limits - compiled contracts data & deployment information for an older version with no limits
+    - deploy - deployment & verification scripts
+    - tests
+        - fixtures - fixtures used by all tests
+        - property - property tests for core contracts
+        - unit - unit tests for internal functions in core contracts
+        - utils - specific utilities for tests, closely related to the contracts logic
+        - main test files for both core & service contracts
```

Comment:
Is there a `-` missing after `main`?

Reply:
Can you please comment on this or fix it?

Reply:
no need for a `-`

Reply:
As you see, there is no `main` folder.

Reply:
Ah, not I get it. Thanks.